app/components/Chart/GlobalEventsEvolution.vue

```
@@ -341,30 +348,15 @@ const visibleLandmarkByIndex = computed(() => {
-                          stroke-linecap="round"
-                          fill="none"
-                        />
-                      </g>
+                      <g
+                        transform="translate(-7.68, -7.68) scale(0.64)"
+                        stroke="currentColor"
+                        stroke-width="2"
+                        stroke-linecap="round"
+                        stroke-linejoin="round"
+                        fill="none"
+                        v-html="landmark.iconSvg"
```

ESLint

[error] Using v-html on component may break component's content.

(vue/no-v-text-v-html-on-component)